### ISSUE TITLE
Can't get editor instance to work with TS and function component, a proper example would be great

### ISSUE THREAD

**Issue by desmap** [securingsincity/react-ace#880]
# Problem

TS reports `Property 'editor' does not exist on type 'never'.`

I have...
```
const aceEditor = useRef(null)
...
useEffect(() => {
  aceEditor.current!.editor.gotoLine(10)
})
...
<AceEditor
  ref={aceEditor}
...
/>
```
re a code sample, I couldn't get react-ace running in CodeSandbox.

**Comment by desmap** [comment]
@securingsincity guess you're busy but  I am about to trash all the work invested in react-ace and replace it with monaco. This is nuts that I just can't access the editor instance.

If this lib is still maintained, it should work with Typescript and function components *flawlessly* in 2020. Looking fwd to your feedback, I hope for a working example!

**Comment by desmap** [comment]
for others driving by, did it not with refs but with an odd work-around:

```
const [instance, setInstance] = useState<Ace.Editor>()
...
return (
  <AceEditor
  onLoad(instance => setInstance(instance))
  ...
```

**Comment by rmariuzzo** [comment]
Hello @desmap! Using your code and some adjustments I can see it [working in this code sandbox](https://codesandbox.io/s/react-ace-gotoline-test-qjyte?file=/src/App.tsx:178-703).

First, I tested your version and it worked after I refreshed the code sandbox (sometimes their cache produce unexpected results). After that, I adjusted your code passing the ref as a dep of the `useEffect` hook and I assert that it is not null with an `if` statement (personal preference over `!`.
